### PR TITLE
connect-azaccount supports ctrl-C

### PR DESCRIPTION
**Pull request by isra-fel**
## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/Accounts/Accounts/Account/ConnectAzureRmAccount.cs [hunk 1]
```diff
@@ -223,6 +222,13 @@ protected override void BeginProcessing()
             AzureSession.Instance.TryGetComponent(WriteWarningKey, out _originalWriteWarning);
             AzureSession.Instance.UnregisterComponent<EventHandler<StreamEventArgs>>(WriteWarningKey);
             AzureSession.Instance.RegisterComponent(WriteWarningKey, () => _writeWarningEvent);
+
+            // todo: ideally cancellation token should be passed to authentication factory as a parameter
+            // however AuthenticationFactory.Authenticate does not support it
+            // so I store it in AzureSession.Instance as a global variable
+            // todo: CancellationTokenSource should be visiable only in cmdlet class
+            // CancellationTokenSource.Token should be passed to other classes
+            AzureSession.Instance.RegisterComponent("LoginCancellationToken", () => new CancellationTokenSource());
```
Comment (isra-fel) [inline src/Accounts/Accounts/Account/ConnectAzureRmAccount.cs:231]: todo: make "LoginCancellationToken" a const
```suggestion
            AzureSession.Instance.RegisterComponent("LoginCancellationToken", () => new CancellationTokenSource());
```

Comment (erich-wang) [inline src/Accounts/Accounts/Account/ConnectAzureRmAccount.cs:231]: Please add third parameter 'true' to force override.

Comment (isra-fel) [reply src/Accounts/Accounts/Account/ConnectAzureRmAccount.cs]: Fixed. Thanks!

**Review by erich-wang — APPROVED** [review]

**Pull request merged**